EIPS/eip-x.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: XXXX
```

Comment:
```suggestion
eip: 7749
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.